app/controllers/mailing_list/steps_controller.rb

```diff
@@ -9,6 +9,7 @@ class StepsController < ApplicationController
     before_action :noindex, unless: -> { request.path.include?("/name") }
     before_action :set_step_page_title, only: %i[show update]
     before_action :set_completed_page_title, only: [:completed]
+    before_action :set_breadcrumb, only: [:completed]
```

Comment:
@martyn-w Just checking that this breadcrumb is in line with how we set it on other pages?

We haven't had a breadcrumb on this page before, and I think it is right to include it - but using the title (that appears in the browser tab and needs to be this way for accessibility) means it is very long

However, if this is consistent with how we generate breadcrumbs elsewhere, I'm happy to leave it as is!

cc @Tamara-Lewis 

<img width="583" alt="image" src="https://github.com/user-attachments/assets/82a4d0fa-3fde-41dc-91b0-56bf4eb718d1" />

Reply:
Hi @gemmadallmandfe, 
yes, we have used a similar approach to generating the breadcrumb trail in the sitemap.

We can of course change the breadcrumb text to be something more succinct if the page title is too wordy?

Reply:
@emmaachesongray @Tamara-Lewis I will leave this decision to you two as to whether this is fine as is or needs amending (quite verbose for screen reader users but may be good to be consistent, so up to you)

Reply:
NB please just check that Marios is happy with this before merging as he may have found an issue